bundles/org.openhab.binding.mqtt.generic/src/main/feature/feature.xml

```diff
@@ -1,12 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<features name="org.openhab.binding.mqtt.generic-${project.version}" xmlns="http://karaf.apache.org/xmlns/features/v1.4.0">
-    <repository>mvn:org.openhab.core.features.karaf/org.openhab.core.features.karaf.openhab-core/${ohc.version}/xml/features</repository>
+<features name="org.openhab.binding.mqtt.generic-${project.version}"
+	xmlns="http://karaf.apache.org/xmlns/features/v1.4.0">
```

Comment:
please make sure that this is on one line. otherwise concatenation might fail when building the full feature-file.